Scripts/Global/NPCTopicsMercGuildmaster.lua

```diff
@@ -0,0 +1,235 @@
+--[[
+Mercenary Guildmaster Topic Template,
+Author: Henrik Chukhran, 2022 - 2024
+]]
+
+-- @note Declare QuestNPC before usage 
+local A, B, C, D, E, F = 0, 1, 2, 3, 4, 5
+local Q = vars.Quests
+
+local function SetBranch(t)
+	QuestBranch(t.NewBranch)
+end
+
+local function Merc_NPCTopicGuildmasterDeclare()
+
+	GuildMasterNotEnoughGoldStr = "It seems your pockets aren't deep enough for my services. Come back when you've got the coin. Quality defense doesn't come cheap, after all."
+
+	NPCTopic{
+		Slot        	=   D,
+		Branch      	=   "",
+		NewBranch   	=   "MercsBrowse0",
+		Ungive      	=   function(t)
+								if #vars.MercNPCAvailableList > 0 then
+									SetBranch(t) 
+								else
+									Message("It appears we are currently short on mercenaries. Come back later.")
+								end
+							end,
+		Texts 			= 
+		{		
+			Topic 		= 	"Browse: Mercenaries",
+			Ungive 		= 	"Of course. We've got a range of skilled fighters, each with their own specialties. Take your pick, but remember - quality comes at a price",
+		},
+	}
+
+	Quest{
+		Slot 			= 	E,
+		Branch			=	"",
+		Texts 			= 	{
+			Topic 		= 	"Find and Return Mercenaries (100g)",
+			Done		=	"If any of your mercenaries are missing, just let me know. I'll ensure they're found and returned to your side promptly, no matter their specialty.",
+			Undone		=	GuildMasterNotEnoughGoldStr
+		},
+		NeverDone       =   true,
+		NeverGiven      =   true,
+		QuestGold		=	100,
+		CanShow			=	function(t) 
+								return #vars.MercNPCHiredList > 0
+							end,
+		Done			= 	function(t) 
```

Comment:
More weird formatting, the text is literally in the middle of my screen, while its not nested that many times.